### PR TITLE
#21: enhancing accessibility

### PR DESCRIPTION
**Pull request by kkoskelin**
presenting 3 document uploads as a list
presenting a single button element whose attributes change, instead of swapping between two buttons for screenreaders
adding aria roles and labels
adjusting color of disabled button for this form
adding aria-live attributes on success message

### CODE REVIEW

**Review by codyseibert — APPROVED** [review]

File: web-client/src/components/FilePetition.jsx [hunk 1]
```diff
@@ -31,86 +31,88 @@ export default connect(
...
-                });
-              }}
-            />
+          <div role="list">
+            <div role="listitem" className="usa-form-group">
+              <label
+                htmlFor="petition-file"
+                className={petition.petitionFile.file && 'validated'}
+              >
+                1. Petition file (form #)
```
Comment (sadlerw) [inline web-client/src/components/FilePetition.jsx:40]: Jessica said this is form #2 it's just lumped with the others on the site now

File: web-client/src/components/FilePetition.jsx [hunk 2]
```diff
@@ -31,86 +31,88 @@ export default connect(
...
+                htmlFor="statement-of-taxpayer-id"
+                className={
+                  petition.statementOfTaxpayerIdentificationNumber.file &&
+                  'validated'
+                }
+              >
+                3. Statement of taxpayer identification number (form #4)
+              </label>
+              <span>
+                To submit your Social Security number or employee identification
```
Comment (sadlerw) [inline web-client/src/components/FilePetition.jsx:88]: Someone should check the language here - should it just be taxpayer identification number?

**Review by sadlerw — APPROVED** [review]
add screenshot please @kkoskelin ?

**Comment by kkoskelin** [comment]
screen shot 2018-10-30 at 2 41 29 pm

**Pull request merged**